examples/01-blog/Blog/Type/Scalar/UrlType.php

```diff
@@ -48,7 +48,7 @@ public function parseValue($value)
      * @return null|string
```

Comment:
also can move this to return type if we're adding types already :)

Reply:
No can do; PHP will complain about violating the `parseLiteral` signature defined in `LeafScalar`.